.editorconfig

```diff
@@ -0,0 +1,24 @@
+; This file is for unifying the coding style for different editors and IDEs.
```

Comment:
I added this because I was opening the project with 2 different IDEs and they kept messing with each other.

Reply:
Wait, are these configurations what's causing the documentation format changes? How do we force that they're being applied without some type of pre-commit or lint checks?

Reply:
Most modern IDEs will automatically read the .editorconfig and apply the style changes without much user input. https://editorconfig.org/